internal/errors/errors.go

```diff
@@ -0,0 +1,15 @@
+package errors
+
+import "strings"
+
+func ErrorFormatter(errs []error) string {
```

Comment:
package name can be `error` and  can be called `error.Formatter` instead of `errors.Formatter`?

Reply:
`error` will conflict with builtin keyword error.

Reply:
ok then `errors.Formatter` still better